norddrop/src/ffi/mod.rs

```diff
@@ -39,6 +39,31 @@ extern "C" {
 #[allow(non_camel_case_types)]
 pub struct norddrop(Mutex<NordDropFFI>);
 
+/// @brief Initialize a new transfer with the peer and descriptors provided and
```

Comment:
What kind of syntax is this? Are you aware that rust uses markdown for doc comments?

Reply:
doxygen

Reply:
Reading https://doc.rust-lang.org/rust-by-example/meta/doc.html now it makes sense why it only supports ///.